packages/react/src/components/iconCard/IconCard.tsx

```diff
@@ -114,18 +125,34 @@ export const IconCard: React.FunctionComponent<IconCardProps & Omit<React.HTMLPr
                             'mod-small': !!small,
                         })}
                     >
-                        {choices.map(({icon, label, value, disabled: choiceDisabled}: IconCardChoice) => (
-                            <li key={value} className="icon-card-drawer-choice">
-                                <button
-                                    className={classNames('inline-flex center-align link', {disabled: choiceDisabled})}
-                                    onClick={() => onClick?.(value)}
-                                    disabled={choiceDisabled}
-                                >
-                                    {icon ? <Svg svgClass="icon mod-24 mod-stroke mr1" svgName={icon} /> : null}
-                                    {label}
-                                </button>
-                            </li>
-                        ))}
+                        {choices.map(
+                            ({
+                                icon,
+                                svgChild: choiceSvgChild,
+                                label,
+                                value,
+                                disabled: choiceDisabled,
+                            }: IconCardChoice) => (
+                                <li key={value} className="icon-card-drawer-choice">
+                                    <button
+                                        className={classNames('inline-flex center-align link', {
+                                            disabled: choiceDisabled,
+                                        })}
+                                        onClick={() => onClick?.(value)}
+                                        disabled={choiceDisabled}
+                                    >
+                                        <SvgChild
+                                            {...({
+                                                svgChild: choiceSvgChild,
+                                                svgName: icon,
+                                                svgClass: 'icon mod-24 mod-stroke mr1',
+                                            } as SvgChildProps)}
+                                        />

```

Comment:
Does it work like this? I find it easier to read

```suggestion
                                        <SvgChild
                                            svgChild={choiceSvgChild}
                                            svgName={icon}
                                            svgClass="icon mod-24 mod-stroke mr1"
                                        />
```